convert2rhel/backup/__init__.py

```diff
@@ -110,12 +111,34 @@ def pop_all(self):
             # logger.critical in some places.
             except (Exception, SystemExit) as e:
                 # Don't let a failure in one restore influence the others
-                loggerinst.warning("Error while rolling back a %s: %s" % (restorable.__class__.__name__, str(e)))
+                message = "Error while rolling back a %s: %s" % (restorable.__class__.__name__, str(e))
+                loggerinst.warning(message)
+                # Add the rollback failures to the list
+                self._rollback_failures.append(message)
 
             processed_restorables.append(restorable)
 
         return processed_restorables
 
+    @property
+    def rollback_failed(self):
+        """
+        Flag of fail when restoring one or more restorables.
+
+        :returns: Boolean value if the rollback failed.
+        """
+
+        return bool(self._rollback_failures)
```

Comment:
Since this is a list I'd prefer changing it to [`any()`](https://docs.python.org/3/library/functions.html#any)
```suggestion
        return any(self._rollback_failures)
```

Reply:
I think this will have the same effect here, no? Both `any()` and `bool()` would return true in case of having at least one element in the list. Especially because the list is a list[str] and not a list[bool]. 

In any case, not against changing to use `any` here, just think the idea will be the same with `bool` and `any`. 

On the other hand, I would expect seeing an `any` check here in case we would have something like `[False, False, True]`, in this case, `any` would be applicable perfectly here, as we care about the only one `True` and not the others `False` inside the list.

Reply:
For sure seems to be the same. I don't have to think "how does arrays get evaluated as a bool?" when I see an `any()` so I prefer that. Just personal preference for readability